lrrbot/commands/live.py

```diff
@@ -41,7 +41,7 @@
 	old_channels = {channel["channel"]["name"] for channel in follows}
 	old_channels.add(config["channel"])
 
-	futures = [twitch.follow_channel(channel, token) for channel in channels.difference(old_channels)]
+	futures = [twitch.follow_channel(channel) for channel in channels.difference(old_channels)]
```

Comment:
This now does `SELECT * FROM users WHERE name = 'lrrbot'` in a loop. The result is unlikely to change while the loop is running.